stripe/src/main/java/com/stripe/android/view/PaymentMethodsAdapter.kt

```diff
@@ -23,13 +23,12 @@ import com.stripe.android.model.PaymentMethod
 internal class PaymentMethodsAdapter constructor(
     private val intentArgs: PaymentMethodsActivityStarter.Args,
     private val addableTypes: List<PaymentMethod.Type> = listOf(PaymentMethod.Type.Card),
-    initiallySelectedPaymentMethodId: String? = null,
     private val shouldShowGooglePay: Boolean = false,
     private val useGooglePay: Boolean = false
 ) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
 
     internal val paymentMethods = mutableListOf<PaymentMethod>()
-    internal var selectedPaymentMethodId: String? = initiallySelectedPaymentMethodId
+    internal var selectedPaymentMethodId: String? = null
```

Comment:
I think the changes to this class can be reverted. If you need to update `selectedPaymentMethodId` you can do so directly (`adapter.selectedPaymentMethodId = ...`) rather than needing to go through `setPaymentMethods()`.